app/frontend/Calendar/DisplayEvent/InvitationDisplay.svelte

```diff
@@ -6,7 +6,6 @@
       {/if}
     </hbox>
     <hbox flex />
-    <AccountDropDown selectedAccount={calendar} accounts={appGlobal.calendars} withIcon={true} withLabel={false} />
```

Comment:
The user can select which calendar to put the event in. That is particularly needed for IMAP account in combination with multiple local or CalDAV calendars.

Reply:
It's not gone, it just got moved somewhere where it was easier for me to populate it correctly.